added support of associated files for structure elements

AlexandrKozlovskiy opened the pull request:
I added support of associated files for structure elements. In future probably it will be need add the same for mc elements,but in this case pdf object should be added on the page,for example via lua code as pdf.setpageresources("/Properties << /P1 <</AF ["..n2.." 0 R] >> >>"),where n2 it dictionary file object,which reference on stream object. Now exists the problem of insertion latex commands inside the associated file. You can test associated files in https://pdf2data.duallab.com/responsivepdf-dev/. Some text shows incorrect,when pdf compile in pdflatex,and we know about this issue. But formulas in my opinion fully accessible with nvda.

u-fischer commented on source/dtx/tagpdf-struct.dtx:
```diff
@@ -287,6 +287,24 @@
+       \prop_if_in:cnT
+     { g__tag_struct_#1_prop }{AF}{
+%why stream not require args expanding?
+\pdf_object_new:nn {mystream\int_eval:n {#1}}{stream}
```
name like "mystream", "mydict" and formula shouldn't go in a general style.

u-fischer commented on source/dtx/tagpdf-struct.dtx:
```diff
@@ -98,7 +98,7 @@
     E,                 %abreviation
     ActualText,
-    AF,                 %pdf 2.0, array of dict, associated files
+    AF,afmime,afrel                 %pdf 2.0, array of dict, associated files
```
afmime and afrel don't belong here, this sequence contains only documented names from the pdf specification.

AlexandrKozlovskiy commented:
Now,i hope,all ok,except problem of insertion latex commands in associated file,which i couldn't solve,or something else? In my opinion,if problem of insertion of latex commands will be solved,everybody can use associated files in pdflatex or lualatex.

AlexandrKozlovskiy commented:
Problem with insertion of latex commands in pdflatex/lualatex,seems,was solved. You can se in it in the latest commit.

AlexandrKozlovskiy commented:
In the latest commit I added counter for associated files,which in future possible use in mc tags and fixed multline environment in test example. Now you can test it on pdf2data.duallab.com/responsivepdf-dev/
